### DISCUSSION TITLE
Schedule one time

### DISCUSSION THREAD

**Discussion by abinaravindh** [argoproj/argo-workflows#6907] [Q&A]
Trying to schedule a workflow for single time execution.
I tried cron workflow with specific time and it works fine. But cron work flow remains in argo with next run scheduled after 364 days.
Is there any way that I can schedule a workflow one time. One executed, workflow automatically gets deleted.

**Comment by sarabala1979** [comment]
you can configure `TTLStrategy ` on the workflow for delete.

**Reply by abinaravindh** [reply]
Thanks for the response.
Using TTLStrategy we are able to delete/configure when the workflow should be deleted. 

In our scenario we are using cron workflow with "58 10 12 10 *" as schedule ( October 12 10:58 AM). 
It triggered the workflow on mentioned time and the triggered workflow got deleted as per TTLStrategy.

But the cron workflow still remains in argo with next run mentioned after 364 days( next year same time).
We do not want to keep this cron schedule. 

Is there any way we can auto delete this after one run or any other way to schedule a job one time for a future date.
